scripts/tools/tools-schema.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
I know that's POC, but please format that file 🩸 👁️

Reply:
Can you elaborate, which file I need to format? I couldn't get it.

Reply:
I think he wants you to run in through pretiffier